src/scenarios/blazorpizza/src/Directory.Build.props

```diff
@@ -1,8 +1,12 @@
 <Project>
-    <PropertyGroup>
+    <PropertyGroup Condition="'$(TargetFramework)' == 'net8.0'">
+        <AspNetCoreVersion Condition="'$(AspNetCoreVersion)' == ''">8.0.0-*</AspNetCoreVersion>
+        <BlazorVersion Condition="'$(AspNetCoreVersion)' == ''">8.0.0-*</BlazorVersion>
+        <SystemNetHttpJsonVersion>8.0.0-*</SystemNetHttpJsonVersion>
+    </PropertyGroup>
+    <PropertyGroup Condition="'$(TargetFramework)' != 'net8.0'">
```

Comment:
Is there a reason to not set net7.0 correctly here and in release/7.0 as well and maybe add a warning for net9?

Reply:
I don't know what the existing cases are, or what depends on these older versions, so I went with preserving older behavior except for the cases that I know about.

If there isn't anything depending on this, then we can set it to net7.0, and the blazor, and STJ versions to match.